package-lock.json

```diff
@@ -0,0 +1,12106 @@
+{
```

Comment:
I was hoping to omit a package-lock from the repository, if that's cool!  I still think projects should utilize one, but I also think it makes sense to have a completely fresh installation with the latest patch versions of everything upon starting a new project.

Reply:
Ohp! My bad! Didn't realize that and that makes sense to me